decidim-core/app/views/decidim/amendments/_wizard_header.html.erb

```diff
@@ -0,0 +1,20 @@
+<div class="flex justify-center">
+  <h1 class="h1 decorator my-12">
+    <%= wizard_header_title %>
+    <%= "(#{similar_resources_count})" if local_assigns[:similar_resources_count] %>
+  </h1>
+</div>
+
+<ol id="wizard-steps" class="wizard-steps">
+  <% (1..total_steps).each do |wizard_step| %>
+    <%
+      scope = "decidim.amendments.wizard_step_form.steps"
```

Comment:
This scope assignment could be taken out of the loop, or passed directly in the `I18n.t`